Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py

```diff
@@ -433,6 +429,11 @@ class AutoInstall(object):
     DISABLE_ENV_VAR = 'DISABLE_WEBKITCOREPY_AUTOINSTALLER'
     CA_CERT_PATH_ENV_VAR = 'AUTOINSTALL_CA_CERT_PATH'
 
+    # This list of libraries is required to install other libraries, and must be installed first
+    BASE_LIBRARIES = ['setuptools', 'wheel', 'pyparsing', 'packaging', 'setuptools_scm']
```

Comment:
Install order is changing from `wheel`, `packaging`, `setuptools` to `setuptools`, `wheel`, `packaging`. Is that intentional?

Reply:
That is intentional. `wheel` needs `setuptools`, `packaging` needs `pyparsing`